app/models/attachment.rb

```diff
@@ -19,7 +19,7 @@ class Attachment < ApplicationRecord
   validates :title, presence: true, length: { maximum: 255 }
   validates :isbn, isbn_format: true, allow_blank: true
   validates :command_paper_number, format: {
-    with: /\A(#{VALID_COMMAND_PAPER_NUMBER_PREFIXES.join('|')}) ?\d+/,
+    with: /\A(#{VALID_COMMAND_PAPER_NUMBER_PREFIXES.map { |prefix| prefix.gsub(/\./, "\\.") }.join('|')}) ?\d+/,
```

Comment:
Can we use `Regexp.escape` here instead?

https://ruby-doc.org/core-2.7.0/Regexp.html#method-c-escape

Reply:
Nice! 👍